packages/cozy-scripts/config/webpack.config.duplicates.js

```diff
@@ -0,0 +1,7 @@
+'use strict'
+
+const DuplicatePackageCheckerPlugin = require("duplicate-package-checker-webpack-plugin");
```

Comment:
```suggestion
const DuplicatePackageCheckerPlugin = require('duplicate-package-checker-webpack-plugin')
```

This makes CI red it seems. But this should be auto-fixed actually